rules/windows/process_creation/proc_creation_win_susp_conhost.yml

```
@@ -46,8 +46,9 @@ detection:
         Image: 'C:\Windows\System32\conhost.exe'
     filter_image_conhost2:
         ParentCommandLine:
-            - '\??\C:\WINDOWS\system32\conhost.exe 0x4'
-            - '\??\C:\WINDOWS\system32\conhost.exe 0xffffffff -ForceV1'
+            # The "?" needs to be escaped as they considered special characters in SIGMA (https://github.com/SigmaHQ/sigma-specification)
+            - '\\?\?\C:\WINDOWS\system32\conhost.exe 0x4'
+            - '\\?\?\C:\WINDOWS\system32\conhost.exe 0xffffffff -ForceV1'
```

```suggestion
            - '\\\?\?\C:\WINDOWS\system32\conhost.exe 0x4'
            - '\\\?\?\C:\WINDOWS\system32\conhost.exe 0xffffffff -ForceV1'
```
the 1st `\` escapes the 2nd one. So we need 3 `\`s to get a literal `\?`.